### ISSUE TITLE
rails Makefile is broken

### ISSUE THREAD

**Issue by ning-y** [source-academy/sicp#211]
On a freshly cloned repository, there is no such folder `./docs`, so this *rm* command will fail. `rm -rf` should work, or `rm -r | true`. 

https://github.com/source-academy/sicp/blob/c9c704f3f1463a904b32a3b3800dd30b7e27b38c/rails/Makefile#L23

The rails script *app/helpers/import_textbook.rb* takes as first argument the absolute path to the directory containing the *xml* directory. The *shell* function was clearly meant to retrieve this absolute path, but it makes a mistake by referring to $(PWD), which refers to the directory containing this Makefile (and therefore is not changed by the *cd*).

Admittedly, $(PWD) could behave differently across GNU Make versions. I'm running 4.1. Anyways, `$(shell cd .. && pwd)` should work fine.

https://github.com/source-academy/sicp/blob/c9c704f3f1463a904b32a3b3800dd30b7e27b38c/rails/Makefile#L19

**Comment by martin-henz** [comment]
Yes, still works for me. We are getting rid of the rails version very soon, so no need to spend time on it. I'm keeping it until the assessments for CS1101S are over, because they may need the textbook with permalinks etc.

**Comment by ning-y** [comment]
Ah, I misunderstood. I thought that the current team's work modifies the static HTML generated by the rails toolchain. Nevermind then, I agree that it's not worth the effort.

**Comment by martin-henz** [comment]
No we get rid of Ruby on Rails! Generation time down from 90 seconds to like 3 seconds. And system complexity just imploded.
